src/wp-content/themes/twentynineteen/style-editor.css

```diff
@@ -583,8 +583,10 @@ h6:lang(vi), figcaption:lang(vi),
 /** === Editor Frame === */
 body .wp-block[data-align="full"],
 body .wp-block.alignfull {
-  width: calc(100% + 20px);
-  max-width: calc(100% + 20px);
+  max-width: calc(100% + 16px);
+  padding-left: 8px;
+  padding-right: 8px;
```

Comment:
I think this is adding extra space around the container now, so full-width isn't actually edge to edge:

<img width="642" alt="Screen Shot 2021-06-22 at 2 51 53 PM" src="https://user-images.githubusercontent.com/541093/122982952-77cecb00-d369-11eb-9a41-6bc1f73ea5b7.png">